backend/server.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const products = require('./routes/products');
+const fakeData = require('./routes/fakeData');
+
+mongoose.connect('mongodb://localhost/products');
```

Comment:
Stephen, your submission is excellent and this review and comments are just for your learning! 
pinging Brian as indeed as well in case he thinks it should be added to the program

@BrianJenney !